tools/oeedger8r/Emitter.ml

```diff
@@ -493,11 +493,6 @@ let gen_enclave_code (ec : enclave_content) (ep : edger8r_params) =
         printf
           "Warning: Function '%s': Reentrant ocalls are not supported by Open \
            Enclave. Allow list ignored.\n"
-          f.uf_fdecl.fname ;
-      if f.uf_is_switchless && not ep.experimental then
```

Comment:
I think we should probably keep switchless ocalls in the experimental stage, and emit a failure if not `--experimental` still.

Reply:
I agree. Though I think by the time switchless call is shipped (target 0.7), we should remove the experimental label.

Reply:
@anakrish and I had wanted all edger8r features (and indeed, most features) to go through one release in `--experimental`, so that they're thoroughly tested. So we should target that in v0.8.

Reply:
@radhikaj What do you think?

Reply:
We will ship switchless calls in V0.7.   We should drop the experimental by then if it is sufficiently tested. We cannot target V0.8 .

Reply:
That means we will skip a beta release cycle for the feature 😬 that scares me!